comskip.c

```diff
@@ -700,7 +700,7 @@ unsigned int		min_black_frames_for_break = 1;
 bool				detectBlackFrames;
 bool				detectSceneChanges;
 int             dummy1;
-unsigned char*		frame_ptr;
+unsigned char*		frame_ptr = 0;

```

Comment:
I got a good explanation of what's going on: https://twitter.com/michaelmelanson/status/873679077764468741

Basically this is masking the original problem by moving this particular variable from BSS to DATA. We need to find where the original out-of-bounds access is occurring which is corrupting memory in the first place.